public/gameList.php

```diff
@@ -198,7 +198,13 @@ function ListGames(
                     echo "<br/>";
                 }
             } else {
-                echo "<h2>$consoleName</h2>";
+                $consoleIconName = str_replace("/", "", $consoleName); // Clean "Mega Drive / Genesis", etc.
+                $fallBackConsoleIcon = "assets/images/system/Unknown.png";
+                echo "<h2 class='flex'>";
+                echo " <img src='assets/images/system/$consoleIconName.png' class='mr-2' alt='Console icon' width='32' height='32'";
```

Comment:
nit: Given this is a `flex` layout, spacing can be controlled with a `gap` utility rather than explicit margin.

```
<h2 class="flex gap-x-2">
```

Reply:
Amended!